axelrod/strategy_transformers.py

```diff
@@ -228,11 +297,21 @@ def dual_wrapper(player, opponent, proposed_action):
     -------
     action: an axelrod.Action, C or D
     """
+
     if not player.history:
-        player.original_player = player.original_class(**player.init_kwargs)
+        player.use_history = []
```

Comment:
Correct, the internal state of the original player is what's important, which means all internal variables must also be retained. Another option is that it may be possible to 'rebuild' the original player from the history.

I agree that some tests definitely need to be included that ensure this.